tools/integration_tests/write_large_files/write_large_files_test.go

```diff
@@ -16,23 +16,44 @@
 package write_large_files
 
 import (
+	"fmt"
 	"log"
 	"os"
 	"testing"
 
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/mounting/static_mounting"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
 )
 
+func compareFileFromGCSBucketAndMntDir(gcsFile string, mntDirFile string, localFilePathToDownloadGcsFile string) (err error) {
+	err = operations.DownloadGcsObject(gcsFile, localFilePathToDownloadGcsFile)
+	if err != nil {
+		err = fmt.Errorf("Error in downloading object:%w", err)
+		return
+	}
+
+	diff, err := operations.DiffFiles(mntDirFile, localFilePathToDownloadGcsFile)
```

Comment:
Nit: DiffFiles loads the entire files into memory. These are both 500 MiB files, hence would have a 1 GiB requirement just for this step.

Reply:
So what changes you are suggesting?

Reply:
No solution at the moment. Just wanted to point this out to know the high memory requirement for this operation.

Reply:
May be capture this in a comment for now, and we can address this later if it causes an issue.